contracts/BSCValidatorSet.sol

```diff
@@ -838,6 +841,48 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     return voteAddrs;
   }
 
+  function setpreviousVoteAddrSet() private {
+    uint n = previousVoteAddrSet.length;
+    uint m = validatorExtraSet.length;
```

Comment:
seems wrong? as the `validatorExtraSet ` is still old.

Reply:
it‘s by design